Automattic/WP-Job-Manager
# Fix/issue 1960

Fixes #1960 

#### Changes proposed in this Pull Request:
* Allows the user to select the pagination type (show more button or pagination links) for the [jobs] page

#### Testing instructions:
* Go to WPJM Settings; to the Job Listings tab
* You should see a new setting (the second one) that allows you to select the pagination type
